validator/client/validator.go

```diff
@@ -639,6 +642,38 @@ func (v *validator) AllValidatorsAreExited(ctx context.Context) (bool, error) {
 	return true, nil
 }
 
+// OneValidatorActive informs if at least one validator is ACTIVE.
+func (v *validator) OneValidatorActive(ctx context.Context) (bool, error) {
```

Comment:
I create this function, but I was thinking of combining it with `AllValidatorsAreExited`?
Unless you guys want specifically the ability to detect and output that validators have been exited.